src/styles/annotator/highlights.scss

```diff
@@ -38,70 +58,53 @@
 .hypothesis-svg-highlight {
   --highlight-color: var(--hypothesis-highlight-color);
   --highlight-text-color: var(--hypothesis-highlight-text-color);
-  --highlight-blend-mode: var(--hypothesis-highlight-blend-mode);
-  --highlight-decoration: var(--hypothesis-highlight-decoration);
   --highlight-color-focused: var(--hypothesis-highlight-focused-color);
 
   & .hypothesis-highlight {
-    --highlight-color: var(--hypothesis-highlight-second-color);
+    --highlight-color: var(--hypothesis-highlight-secondColor);
 
     .hypothesis-highlight {
       // Highlights more than two levels deep are transparent by default.
-      --highlight-color: var(--hypothesis-highlight-third-color);
+      --highlight-color: var(--hypothesis-highlight-thirdColor);
     }
   }
 }
 
-// Configure clustered highlight styling. The `.hypothesis-highlights-clustered`
-// class is managed by `highlight-clusters`
-
-.hypothesis-highlights-clustered .hypothesis-highlight {
-  --highlight-text-color: var(--hypothesis-cluster-text-color);
-}
-
-.hypothesis-highlights-clustered .hypothesis-highlight,
-.hypothesis-highlights-clustered .hypothesis-svg-highlight {
-  // When clustered highlights are active, use an opaque blue for focused
-  // annotations so we don't end up with a funny color mix
-  --highlight-color-focused: var(--hypothesis-color-blue);
-
-  &.user-annotations {
-    --highlight-color: var(--hypothesis-user-annotations-color);
-    --highlight-decoration: var(--hypothesis-user-annotations-decoration);
+@mixin clusterHighlightStyles($clusterValue) {
```

Comment:
`$clusterValue` seems more like a key rather than a value. In my head I think of the logical group as a key/name, which is then associated with a number of color values.

Reply:
What you're observing here is me tripping over myself because a lack of naming clarity. This particular prototype is chasing down the styling of clusters "to do with" annotation ownership. I anticipate we may in the future "cluster" by different criteria, for example an explicit highlight-pen color set by the user on specific annotations, or some other "baked" clusters (i.e. the set of available cluster values for the cluster group — there I am using vague terminology again — are known to the app at runtime, c.f. styling highlights by a dynamic value, say, a tag or something).

I anticipate that we would only activate one cluster...group?... at a time, that is, you could style highlights based on their "ownership" cluster, _or_ you could style highlights based on their "pen color" cluster. Not both at the same time.

So, we're going to want to play a bit of a naming game sometime soon. What does _cluster_ mean? etc. I will try to make this SASS mixin more clear.